unison-src/transcripts/delete.output.md

```diff
@@ -182,10 +182,6 @@ type Foo = Foo Boolean
     5. ┌ Foo.Foo#d97e0jhkmd#0 : Nat -> Foo
     6. └ Foo.Foo#gq9inhvg9h#0 : Boolean -> b.Foo
   
-  Added definitions:
-  
-    7. foo : Nat
-  
```

Comment:
This is the same erroneous undeletion that #1333 also resolves (by avoiding use of the `updateChildren` function in the first place). Nice to see that it also is fixed here by changing how `updateChildren` works.

Reply:
So after this patch you would switch back to updateChildren?

Reply:
Might be a good idea.